haystack/document_stores/pinecone.py

```diff
@@ -684,9 +684,7 @@ def write_documents(
                         embeddings = [embed.tolist() if embed is not None else None for embed in embeddings_to_index]
                     else:
                         # Use dummy embeddings for all documents
-                        embeddings_to_index = np.zeros((len(document_chunk), self.embedding_dim), dtype="float32")
-                        # Convert embeddings to list objects
-                        embeddings = [embed.tolist() if embed is not None else None for embed in embeddings_to_index]
+                        embeddings = [self.dummy_vector] * len(document_chunk)
```

Comment:
@anakin87 This is the only diff (aside from -10.0) right? Everything else is the same, no?

Reply:
yes!